signup.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Signup</title>
+    <link rel="stylesheet" href="signupStyle.css" />
+  </head>
+  <body>
+    <div class="signup">
+      <div class="logo">
+        <img class="logo-image" src="image/logo.png" alt="logo" />
+        <a href="/">
+          <img class="pandamarket-logo" src="image/pandamarket.png" alt="pandamarket" />
+        </a>
+      </div>
+      <div class="signup-form">
+        <div class="signup-box">
+          <div class="email">
+            <h3>이메일</h3>
+            <input type="text" placeholder="이메일을 입력해주세요" />
+          </div>
+          <div class="nickname">
+            <h3>닉네임</h3>
+            <input type="text" placeholder="이메일을 입력해주세요" />
+          </div>
+          <div class="password">
+            <h3>비밀번호</h3>
+            <div class="password-input">
+              <input type="password" placeholder="비밀번호를 입력해주세요" />
+              <img src="image/hidden.png" width="24px" height="24px" alt="" />
```

Comment:
유저의 이벤트를 일으키는 경우 `button` 속성으로 감싸주면 더 시멘틱합니다. 
```suggestion
<button>
              <img src="image/hidden.png" width="24px" height="24px" alt="" />
</button>
```